charts/supabase/templates/db/autoscaling.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if .Values.db.autoscaling.enabled -}}
+apiVersion: autoscaling/v2
+kind: HorizontalPodAutoscaler
+metadata:
+  name: {{ include "supabase.db.fullname" . }}
+spec:
+  scaleTargetRef:
+    apiVersion: apps/v1
+    kind: Deployment
+    name: {{ include "supabase.db.fullname" . }}
+  minReplicas: {{ .Values.db.autoscaling.minReplicas }}
```

Comment:
If I get that right, you removed .Values.db.autoscaling.minReplicas from values.yaml 🤔

Reply:
Yes, since Postgres has no real active-active-cluster mechanism, so Scaling the Database wouldn't create a bernefit.
However there may be alternative Postgres-Compatible Systems which could benefit from Autoscaling, so i removed it from the values but kept it in the template, in case someone using something other than Postgres wants this capability.